Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -519,10 +520,8 @@ bool VolumeVerifier::CheckPartition(const Partition& partition)
   if (partition.offset % VolumeWii::BLOCK_TOTAL_SIZE != 0 ||
       m_volume.PartitionOffsetToRawOffset(0, partition) % VolumeWii::BLOCK_TOTAL_SIZE != 0)
   {
-    AddProblem(
-        Severity::Medium,
-        StringFromFormat(Common::GetStringT("The %s partition is not properly aligned.").c_str(),
-                         name.c_str()));
+    AddProblem(Severity::Medium,
+               fmt::format(Common::GetStringT("The {0} partition is not properly aligned."), name));
```

Comment:
Given this is used quite a bit, maybe this should be its own helper function in common, like:

```cpp
typename <typename... Args>
std::string GetStringTFmt(const char* string, Args&&... args)
{
    return fmt::format(GetStringT(string), std::forward<Args>(args)...);
}
```

So that `fmt::format` doesn't need to be typed all the time in conjunction with another function immediately one after another in several spots.

Reply:
Added. I wasn't sure whether it would fit best in StringUtil.h (because it deals with string formatting) or MsgHandler.h (because it deals with translation), but I put it in StringUtil.h.

Reply:
I moved it to MsgHandler.h because I figured we will need to include fmt/format.h in that header anyway if we migrate panic alerts over to using fmt.

Reply:
Sounds good!